src/components/Dialog/CollectionsDialog.js

```diff
@@ -153,6 +177,12 @@ const CollectionsDialog = ({
       case COLLECTIONS_DIALOG_ACTIONS.DELETE_ERROR:
         setCollectionsDialogText(null);
         break;
+      case COLLECTIONS_DIALOG_ACTIONS.DISCARD:
+        navigation.navigate('CollectionsList');
```

Comment:
This switch case is really just for modifying the contents of the Dialog. Navigation actions should occur in the `onPress` function.

Reply:
will be sure to fix this in a future issue - I tried resolving it and ended up throwing more errors for a little bit - will require a deeper look